src/pages/Register.js

```diff
@@ -105,11 +159,81 @@ const Register = () => {
                 id,
                 value: email,
                 onChange: (v) => setEmail(v),
-                style: { width: '50ex' },
+                style: { width: '66ex' },
+              }),
+            ]),
+          ]),
+      ]),
+      h(IdContainer, [
+        (id) =>
+          div({ style: { lineHeight: '170%' } }, [
+            label({ htmlFor: id, style: { display: 'block', marginTop: '2rem' } }, ['Organization *']),
+            div([
+              h(TextInput, {
+                id,
+                value: institute,
+                onChange: (v) => setInstitute(v),
+                disabled: !partOfOrganization,
+                style: { width: '66ex' },
               }),
             ]),
           ]),
       ]),
+      h(IdContainer, [
+        (id) =>
+          h(div, { style: { lineHeight: '170%', style: { marginTop: '0.25rem' } } }, [
+            h(
+              LabeledCheckbox,
+              {
+                checked: !partOfOrganization,
+                onChange: () => setPartOfOrganization(!partOfOrganization),
+                'aria-describedby': id,
```

Comment:
I don't think the ID linking is correct here. Can you please check the JavaScript console for accessibility errors? I would check, but I can't get to the register page since I already have an account.

Reply:
I did not see any Axe errors in the console.

Reply:
To be clear, you won't see them on the PR site… only when you run locally.

Is there a way I can see the register page?